apiserver/crossmodel/crossmodel.go

```diff
@@ -218,7 +238,30 @@ func makeOfferParamsFromOffer(url string, offer jujucrossmodel.ApplicationOffer)
 // FindApplicationOffers gets details about remote applications that match given filter.
 func (api *API) FindApplicationOffers(filters params.OfferFilters) (params.FindApplicationOffersResults, error) {
 	var result params.FindApplicationOffersResults
+	offers, err := api.getApplicationOffersDetails(filters)
+	if err != nil {
+		return result, err
+	}
+	for _, offer := range offers {
+		result.Results = append(result.Results, offer.ApplicationOffer)
+	}
+	return result, nil
+}
+
+// ListApplicationOffers gets deployed details about application offers that match given filter.
+// The results contain details about the deployed applications such as connection count.
+func (api *API) ListApplicationOffers(filters params.OfferFilters) (params.ListApplicationOffersResults, error) {
```

Comment:
Yes, correct. And also because different access permissions will be in play. You'll need consume access for the Find call, and Admin for the List call.